src/appleseed/foundation/meta/tests/test_aabb.cpp

```diff
@@ -181,80 +170,80 @@ TEST_SUITE(Foundation_Math_AABB)
 
     TEST_CASE(TestOverlapRatio)
     {
-        EXPECT_FEQ(0.0,
+        EXPECT_EQ(0.0,
```

Comment:
I'm not sure why you changed these. Using `EXPECT_FEQ()` to test floating point equality was correct.